### ISSUE TITLE
Visualization of *.vti file

### ISSUE THREAD

**Issue by SimonNgj** [MCInversion/SurfaceEvolverDevelop#3]
How can we visualize distance field in *.vti file? I used VTK: https://kitware.github.io/vtk-examples/site/Cxx/IO/ReadImageData/
to open max-planck_voxFieldSDF30_FSM_30.vti, but I only saw an empty rectangular cube

![image](https://user-images.githubusercontent.com/22513384/185844542-fabdb4ac-590b-40ac-a3af-f6d25d13a608.png)

**Comment by MCInversion** [comment]
Hi, 
*.vti files can be opened by Paraview as volume rendered data:
![Screenshot (21)](https://user-images.githubusercontent.com/34405724/185883699-7f94c9c5-32f3-485d-bf30-49dbb37c39a8.png)
or alternatively as a list of contours:
![Screenshot (22)](https://user-images.githubusercontent.com/34405724/185883775-c2ec3029-58c7-4490-b0c4-a2a43f906a76.png)
If you're working on your own VTK rendering tool, you must play around with the options for `vtkRenderer`. Make sure the data buffer is non-zero after reading from the file.

**Comment by SimonNgj** [comment]
Thanks, but my distance field looks very weird
![image](https://user-images.githubusercontent.com/22513384/185921717-b55b8cc8-eaec-416e-96cf-d6ef0557b876.png)

**Comment by SimonNgj** [comment]
The distance field is only a half, and misaligned with the mesh
![image](https://user-images.githubusercontent.com/22513384/185925758-2e004e7b-217b-492b-99cd-5a8f6f414178.png)

**Comment by MCInversion** [comment]
TLDR; `git pull`. You might have a buggy commit on your machine. Moreover, there were additional improvements I needed to make, e.g.: clearing memory leaks etc.

The `leafSize` was an experimental version of the SDF computation. It got into the master branch accidentally as a mishandled merge conflict. If you take a look at the latest version of SDF.cpp, all of the `Octree` constructors accept only grid resolution. 

I paid for this mistake when I was running experiments while writing [this article](https://cescg.org/cescg_submission/advection-driven-shrink-wrapping-of-triangulated-surfaces/). All of the SDFs were computed super fast which was weird. When I tried visualizing them I only saw a single voxel.

**Comment by SimonNgj** [comment]
Thanks for sharing your interesting paper. The distance field obtained from FastSweep3D is sometimes very weird. All values are inf. It could be something wrong with the grid initialization.

**Comment by MCInversion** [comment]
That's absolutely ok. `setLeafValueToScalarGrid` basically creates an "outline" of the mesh: 
![Fs0](https://user-images.githubusercontent.com/34405724/186100926-1cb59fe1-d0bd-47ef-9305-17e273a35667.jpg)
Since, during construction `Grid` gets initialized with values `LARGE_VAL` everywhere, and `setLeafValueToScalarGrid` writes exact distances to mesh for voxels intersecting the mesh, then if `LARGE_VAL` is not a particular implicitly `double` value (e.g.: `20.0`) for truncation, there should be numerical infinities everywhere else because you need to call `FastSweep3D::sweep` to complete the rest of the field.

If you have a non-infinite value of macro `LARGE_VAL`, there might be something else going on. I'd have to play around with the output.

**Comment by SimonNgj** [comment]
Thank you, may be the version that I cloned has some errors. I'm trying a simple example, but the result does not make sense. Very appreciated if you can let me know what's wrong.
![image](https://user-images.githubusercontent.com/22513384/186162310-01cd31fa-92fe-42e2-99cf-b02ae70ce68b.png)
![image](https://user-images.githubusercontent.com/22513384/186162338-39efbfc9-c13f-4ea5-b21b-f25f4abf7f43.png)

**Comment by MCInversion** [comment]
Quick fix: Change `res` to `40` for example. This means that the original Octree cube will be subdivided into 40x40x40 cells. The relevant cells are then mapped onto `Grid` which is a bit larger to contain the rest of the field. Octree resolution means: into how many pieces an octree root cube is subdivided until it reaches the leaves.
